### PR TITLE
Add GroupDetails UI

### PR DESCRIPTION
**Pull request by NCMoseley**

### CODE REVIEW

File: js/screens/GroupDetails/GroupDetails.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          <Text style={styles.leave}>Leave Group</Text>
+        </TouchableOpacity>
+      </View>
+    </ScrollView>
+  );
+};
+
+export default GroupDetails;
+
+GroupDetails.propTypes = {
```
Comment (siddharthparmar7) [inline js/screens/GroupDetails/GroupDetails.js:61]: if you are using .isRequired you do not need default propType.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Thank you!

**Comment by NCMoseley** [comment]
Ok looking better.

**Review by siddharthparmar7 — APPROVED** [review]
Perfect! thank you!

**Pull request merged**